Makefile

```diff
@@ -4,7 +4,7 @@ PYTHON ?= python
 PYTESTS ?= py.test
 CODESPELL_SKIPS ?= "doc/_build,doc/auto_*,*.fif,*.eve,*.gz,*.tgz,*.zip,*.mat,*.stc,*.label,*.w,*.bz2,*.annot,*.sulc,*.log,*.local-copy,*.orig_avg,*.inflated_avg,*.gii,*.pyc,*.doctree,*.pickle,*.inv,*.png,*.edf,*.touch,*.thickness,*.nofix,*.volume,*.defect_borders,*.mgh,lh.*,rh.*,COR-*,FreeSurferColorLUT.txt,*.examples,.xdebug_mris_calc,bad.segments,BadChannels,*.hist,empty_file,*.orig,*.js,*.map,*.ipynb,searchindex.dat,install_mne_c.rst,plot_*.rst,*.rst.txt,c_EULA.rst*,*.html,gdf_encodes.txt,*.svg,references.bib,*.css,*.edf,*.bdf,*.vhdr"
 CODESPELL_DIRS ?= mne/ doc/ tutorials/ examples/
-all: clean inplace test test-doc
+all: clean test-doc
```

Comment:
attention: These are non-f-string-related changes ☝️ 

These targets do not exist in the Makefile and were giving me errors, so I suggest to remove them.